src/views/Scenario/Scenario.js

```diff
@@ -323,8 +328,9 @@ const Scenario = (props) => {
       </Grid>
       <Card>
         <SimplePowerBIReportEmbed
+          key={currentScenario?.data?.id}
```

Comment:
Do we need to add a key to this component ?

Reply:
Ok, adding a comment to explain why we should keep the `key` might be helpful later :smile: